gui/designwidget.cc

```diff
@@ -528,6 +539,11 @@ void DesignWidget::onClickedPip(PipId pip, bool keep)
     }
     
     int index = getIndexByElementType(ElementType::PIP);
+    if (!keep)
+    {
+        for(int i=0;i<6;i++)

```

Comment:
Coding style is inconsistent with rest of the codebase (space after ';' and before '(').

Reply:
Oh, there's 'clangformat' later in this patchset. Nevermind…